src/server/pfs/server/driver.go

```diff
@@ -3722,20 +3727,22 @@ func (d *driver) getTreeForFile(pachClient *client.APIClient, file *pfs.File) (h
 			return err
 		}
 		if commitInfo.Finished != nil {
-			t, err := d.getTreeForCommit(txnCtx, file.Commit)
-			if err != nil {
-				return err
-			}
-
-			result, err = t.Copy()
+			fmt.Printf("Finished commit\n")
```

Comment:
Sorry for the drive-by comment, but this looks like it might be a stray debug log? (PR seems interesting!)

Reply:
Thanks!